README.md

```diff
@@ -35,3 +39,11 @@ All new development should be done in a branch named `<initials>/<description>`
 	git checkout -b <initials>/<description>
 
 When the feature is complete, tests pass, and you are ready for it to be merged, create a PR.
+
+Pull requests must have at least one approving review (ideally two), and the CircleCI tests must pass.
```

Comment:
This says it requires at least one but the branch permissions requires two, we should probably straighten one of those out.

Reply:
Oh Caleb must have set it back to two. Will fix, good catch.